src/util/time.h

```diff
@@ -36,9 +36,9 @@ class Time {
     // Returns the time elapsed since Mixxx started up in milliseconds.
     static qint64 elapsedMsecs() {
         if (s_testMode) {
-            return s_testElapsedTime / 1000;
+            return s_testElapsed_nsecs / 1000000;
         }
-        return s_timer.elapsed() / 1000;
+        return (s_timer.elapsed() / 1000000);
```

Comment:
Sorry, by "turning this back on" I mean fixing it to actually return milliseconds as you've done.